server/events/project_command_runner.go

```diff
@@ -346,7 +354,15 @@ func (p *DefaultProjectCommandRunner) doApply(ctx models.ProjectCommandContext)
 	}
 	defer unlockFn()
 
+	// Create a PR status to track project's apply status. The status will
+	// include a link to view the progress of atlantis apply command in real
+	// time
+	if err := p.ProjectCmdOutputHandler.SetJobURLWithStatus(ctx, models.ApplyCommand, models.PendingCommitStatus); err != nil {
```

Comment:
We might have to exclude these from mergeable. otherwise that apply requirement will break this. I might have already done that before but we should double check.

Reply:
I'm setting them back to `SuccessCommitStatus`, it should affect `mergeable` condition?

Reply:
yeah but looks like we're fine since we use `startsWith()`: https://github.com/lyft/atlantis/blob/release-v0.17.1-lyft.1/server/events/vcs/github_client.go#L381-L382